pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -183,6 +183,9 @@ type JaegerCommonSpec struct {
 
 	// +optional
 	ServiceAccount string `json:"serviceAccount,omitempty"`
+
+	// +optional
+	HostNetWork bool `json:"hostNetWork,omitempty"`
```

Comment:
This should be `*bool` instead. Nil indicates that users have not set this flag explicitly in the CR.